### ISSUE TITLE
win.blur() doesn't work on linux

### ISSUE THREAD

**Issue by xiaoyu3x** [nwjs/nw.js#5877]
nw.js version: v0.20.1
os version: 14.04
the following is code:

```
var win_1 = nw.Window.get();
var win_2 = null;
nw.Window.open('secondapp.html', {
  "title": "Second App",
  "position": "mouse"
  }, function(w) {
    win_2 = w;
    win_1.focus();
  }
)
function moveFocusAway() {
  win_1.blur();
}
```

win_1.blur() doesn't move focus away to secondapp.html.

**Comment by Christywl** [comment]
I can reproduce this issue on Linux/Windows with nwjs-sdk-v0.22.0.

**Comment by rogerwang** [comment]
I don't think this is a bug, because the button you are using for testing is in win1 , clicking on it takes the focus back after blur.

**Comment by xiaoyu3x** [comment]
@rogerwang 
Why http://docs.nwjs.io/en/latest/References/Window/#winblur describe "Move focus away" .
Could this code be wrong. 
Focus can't be switch to secondwindow.html  if code is "win_2.blur".
Please help me, how to use win.blur.
